javagen/src/main/java/com/azure/autorest/template/example/ProtocolExampleWriter.java

```diff
@@ -203,64 +197,9 @@ public ProtocolExampleWriter(ProtocolExample protocolExample) {
                         // Path cannot be optional
                     }
                 });
-
-                method.getProxyMethod().getAllParameters().stream().filter(ProxyMethodParameter::isFromClient).filter(p -> p.getName().equalsIgnoreCase(parameterName)).findFirst().ifPresent(p -> {
-                    String clientValue = p.getClientType()
-                            .defaultValueExpression(parameterValue.getObjectValue().toString());
-
-                    serviceClient.getProperties().stream().filter(p1 -> Objects.equals(p.getName(), p1.getName())).findFirst().ifPresent(serviceClientProperty -> {
-                        processedServiceClientProperties.add(serviceClientProperty);
-
-                        clientParameterLines.add(
-                                String.format(".%1$s(%2$s)", serviceClientProperty.getAccessorMethodSuffix(), clientValue));
-                    });
-                });
             }
```

Comment:
The biggest refactor here: 
Extracted handling of "proxy method parameters whose values come from client" out of the parameter value assignment loop. 

When the parameter is from client, I think we don't have to look into the method signatures to find it? The value should be set in client instead.

Reply:
Sounds correct.